app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_command = command.split()
+    if list_command[1] == list_command[2] and list_command[0] == "cp":
+        return
+    with (open(list_command[1], "r") as file_in,
+          open(list_command[2], "w") as file_out):
```

Comment:
You can unpack list_command instead of getting value by index. It makes your code clearer.

Reply:
> You can unpack list_command instead of getting value by index. It makes your code clearer.

How to understand?